galaxy_ml/model_validation.xml

```diff
@@ -1,5 +1,5 @@
-<tool id="sklearn_model_validation" name="Model Validation" version="@VERSION@">
-    <description>evaluates estimator performance by cross-validation</description>
+<tool id="sklearn_model_validation" name="Evaluate scikit-learn and deep learning models" version="@VERSION@">
+    <description>performances by cross-validation</description>
     <macros>
```

Comment:
How about `Model Validation, evaluates estimator (including scikit-learn and deep learning) performance`?
I don't have strong preference though.
The same situation applies to searchcv tool, shared by sklearn, deep learning and other kinds of models.